docs/dev/new_check_howto.md

```diff
@@ -487,30 +528,36 @@ After you've created your Agent-based integration, refer to this list to make su
 - A complete `manifest.json` file.
 - If the integration collects Service Checks, the `service_checks.json` must be complete as well.
 
-After you open a pull request in the `integrations-extras` repository, CI validation tests will run. These tests must be green before your pull request is merged and your integration tile is published.
+Before you open a pull request, run the following command to catch any problems with your integration:
+```
+ddev validate all changes
+```
+
+After you've created your pull request, automatic checks run to verify that your pull request is in good shape and contains all the required content to be updated.
 
 ## Further Reading
 
 Additional helpful documentation, links, and articles:
 
-- [Manage integrations via API calls][19]
+- [Manage integrations via API calls][18]
+- [Python for Agent-based Integration Development][3]
 
 [1]: https://docs.datadoghq.com/developers/#creating-your-own-solution
-[2]: https://docs.docker.com/get-docker/
-[3]: https://wiki.python.org/moin/BeginnersGuide/Download
-[4]: https://github.com/pyenv/pyenv
-[5]: https://github.com/DataDog/integrations-extras
-[6]: https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/
-[7]: https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev
-[8]: https://docs.datadoghq.com/developers/metrics/agent_metrics_submission/
-[9]: https://github.com/DataDog/datadog-agent/blob/6.2.x/docs/dev/checks/python/check_api.md
-[10]: https://docs.pytest.org/en/latest
-[11]: https://github.com/pypa/hatch
-[12]: https://datadoghq.dev/integrations-core/meta/config-specs/
-[13]: https://docs.datadoghq.com/developers/integrations/check_references/#configuration-file
-[14]: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
-[15]: https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file
-[16]: https://docs.datadoghq.com/developers/integrations/check_references/#service-check-file
-[17]: https://packaging.python.org/en/latest/tutorials/packaging-projects/
-[18]: https://docs.datadoghq.com/agent/
-[19]: https://www.datadoghq.com/blog/programmatically-manage-your-datadog-integrations/
+[2]: https://github.com/pypa/pipx
+[3]: /developers/integrations/python/
```

Comment:
The relative links won't work on github (but neither will the tabs so maybe that's okay with you)